central/graphql/resolvers/vulnerability_counter.go

```diff
@@ -123,6 +123,25 @@ func mapCVEsToVulnerabilityCounter(fixable, unFixable []*storage.CVE) *Vulnerabi
 	return counter
 }
 
+func mapNodeCVEsToVulnerabilityCounter(fixable, unFixable []*storage.NodeCVE) *VulnerabilityCounterResolver {
```

Comment:
I think this is fine for now, but I'm pretty sure with postgres there's some way of being able to do this query easily with less overhead, something to look at in the future

Reply:
+1

Reply:
+1 Connor